### PR TITLE
Set default values for args in infer

### LINKED ISSUE #1196: Default arg values for infer recipe
## What and why
User should not have to manually set `--optimizer` and `--batch_size` when calling infer.

If the user does not set those when calling from cmd line, errors are thrown.

## Solution(s)
In `parse_args`, if `infer` recipe is used, set the arg values.

## Acceptance criteria
User can call `infer` without setting certain parameters that do not matter:

```bash
PREFIX=$HOME/dropbox/sts-data
GPU=0
BOOTSTRAP=0
./scripts/run.sh -d $GPU $PWD/ml4c3/recipes.py infer \
--num_workers 4 \
--tensors     /storage/shared/ecg/mgh \
--tensors     $PREFIX/sts-mgh.csv sts \
--train_csv   $PREFIX/bootstraps-ecg/$BOOTSTRAP/train.csv \
--valid_csv   $PREFIX/bootstraps-ecg/$BOOTSTRAP/valid.csv \
--test_csv    $PREFIX/bootstraps-ecg/$BOOTSTRAP/test.csv \
--model_file  $HOME/dropbox/ecgnet-sts/results/ecgnet-v011/$BOOTSTRAP/model-weights.h5 \
--remap_layer  input_ecg_2500_zscore_pop_60_days_before_sts_continuous input_ecg_2500_zscore_pop_60_days_before_sts_newest_continuous \
--remap_layer  output_sts_death_60_days_after_ecg_categorical output_sts_death_60_days_after_ecg_newest_categorical \
--remap_layer  output_sts_stroke_60_days_after_ecg_categorical output_sts_stroke_60_days_after_ecg_newest_categorical \
--remap_layer  output_sts_renal_failure_60_days_after_ecg_categorical output_sts_renal_failure_60_days_after_ecg_newest_categorical \
--remap_layer  output_sts_prolonged_ventilation_60_days_after_ecg_categorical output_sts_prolonged_ventilation_60_days_after_ecg_newest_categorical \
--remap_layer  output_sts_dsw_infection_60_days_after_ecg_categorical output_sts_dsw_infection_60_days_after_ecg_newest_categorical \
--remap_layer  output_sts_reoperation_60_days_after_ecg_categorical output_sts_reoperation_60_days_after_ecg_newest_categorical \
--remap_layer  output_sts_long_stay_60_days_after_ecg_categorical output_sts_long_stay_60_days_after_ecg_newest_categorical \
--input_tensors \
    ecg_2500_zscore_pop_60_days_before_sts_newest \
--output_tensors \
    sts_death_60_days_after_ecg_newest \
    sts_stroke_60_days_after_ecg_newest \
    sts_renal_failure_60_days_after_ecg_newest \
    sts_prolonged_ventilation_60_days_after_ecg_newest \
    sts_dsw_infection_60_days_after_ecg_newest \
    sts_reoperation_60_days_after_ecg_newest \
    sts_long_stay_60_days_after_ecg_newest \
--output_folder $HOME/dropbox/@deleteme
```

### PR DESCRIPTION
**Pull request by erikr**
## Issues closed
Closes #1196

## Changelog
- [x] Sets defaults for `--batch_size` when calling `infer`.

## Checklist
- [x] The following are complete:
    - Added or revised tests if appropriate.
    - If new packages are installed, `requirements.txt` is updated.
    - If documentation should be updated, a new issue was made in `ml4c3-wiki`.
    - The PR name describes its content and does not contain "Ref #".
    - The PR does not add PHI.

### CODE REVIEW

**Comment by erikr** [comment]
Tests currently fail because of https://github.com/aguirre-lab/ml4c3/issues/1199.

**Comment by erikr** [comment]
No longer sets default for `optimizer` since that is loaded in the model file.

File: ml4c3/arguments.py [hunk 1]
```diff
@@ -1282,7 +1282,7 @@ def _process_args(args: argparse.Namespace):
     # If infer recipe, set default args
     if args.recipe == "infer":
         args.batch_size = 256
-        args.optimizer = "adam"
+        # args.optimizer = "adam"
```
Comment (StevenSong) [inline ml4c3/arguments.py:1285]:
```suggestion
```

**Review by StevenSong — APPROVED** [review]
looks good, thanks for debugging and implementing, just a suggestion to remove the commented out code

**Pull request merged**